### ISSUE TITLE
HTTPs Settings are not working

### ISSUE THREAD

**Issue by mntolia** [emqx/emqx-management#211]
OS: Ubuntu 18
EMQX:4.2

So I set the https settings in the emqx_management.conf file like this:
```
management.listener.https = 8081
management.listener.https.acceptors = 2
management.listener.https.max_clients = 512
management.listener.https.backlog = 512
management.listener.https.send_timeout = 15s
management.listener.https.send_timeout_close = on
management.listener.https.certfile = etc/certs/cert.pem
management.listener.https.keyfile = etc/certs/privkey.pem
management.listener.https.cacertfile = etc/certs/chain.pem
```
This is the same setting I am using for the dashboard config. It works fine in the dashboard but does not work in the the management.

When I send a GET request to https://url:8081/api/v4/, the server does not give any response. In the dashboard it shows the listener is active.

**Comment by HJianBo** [comment]
Hi @mntolia It seems the default certificate is outdated

We have updated it in v4.0.7 and it will also be released in v4.1-rc.1  

Thanks for your feedback. Plz try it later

**Comment by mntolia** [comment]
Hi @HJianBo ,
I am not using the default. I am using a Letsencrypt certificate. The certificate settings work fine for the dashboard but not for the management plugin.

Thank you.
